### PR TITLE
Tasks 2, 3 with tests

### PR DESCRIPTION
**Pull request by ValeriL**

### CODE REVIEW

File: Homework/homework2/task3.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+def combinations(*args: List[Any]) -> List[List]:
+    """
+    Write a function that takes K lists as arguments.
+
+    Returns all possible lists of K items where the first element is from the first list,
+    the second is from the second and so one.
+    You may assume that that every list contain at least one element
+
+    Combinations are not repeated
```
Comment (Cjkjvfnby) [inline Homework/homework2/task3.py:13]: I don't sure that it is in task.

File: Homework/homework2/task3.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    """
+    comb_tuple = list(itertools.product(*args))
```
Comment (Cjkjvfnby) [inline Homework/homework2/task3.py:15]: Can you solve it without the product?

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]

File: Homework/homework2/task2.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+def major_and_minor_elem(inp: List) -> Tuple[int, int]:
+    """
+    Given an array of size n, find the most common and the least common elements.
+
+    The most common element is the element that appears more than n // 2 times.
+    The least common element is the element that appears fewer than other.
+    You may assume that the array is non-empty and the most common element
+    always exist in the array.
+
+    If there are more than one minor, the function returns the first entered value
```
Comment (Cjkjvfnby) [inline Homework/homework2/task2.py:14]: Don't forget about punctuation.  You need to end the sentence with period.

File: Homework/homework2/task3.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+from typing import Any, List
+
+
+def combinations(*args: List[Any]) -> List[List]:
+    """
+    Write a function that takes K lists as arguments.
+
+    Returns all possible lists of K items where the first element is from the first list,
+    the second is from the second and so one.
```
Comment (Cjkjvfnby) [inline Homework/homework2/task3.py:9]: Be sure that Grazie plugin is installed into the Pycharm

image
image

**Review by Cjkjvfnby — APPROVED** [review]

File: Homework/homework2/task3.py (hunk 4 above)
Comment (ValeriL) [reply Homework/homework2/task3.py:9]: I hadn't this plugin installed, thank you

**Pull request merged**